src/main/java/net/minecraftforge/common/extensions/IForgeBlockState.java

```diff
@@ -648,10 +624,10 @@ default boolean shouldDisplayFluidOverlay(BlockAndTintGetter world, BlockPos pos
      * @return The resulting state after the action has been performed
      */
     @Nullable
-    default BlockState getToolModifiedState(Level world, BlockPos pos, Player player, ItemStack stack, ToolType toolType)
+    default BlockState getToolModifiedState(Level world, BlockPos pos, Player player, ItemStack stack)
```

Comment:
I don't like removing the tool type argument from this. I have some tools that support performing tilling or other interactions without being a hoe for instance, and have even successfully made a tool that can both till and strip wood (if the player makes weird choices). Removing tool type also means I am now stuck relying on the order defined by blocks instead of declaring which interaction I am interested in.